### ISSUE TITLE
Hornet

### ISSUE THREAD

**Issue by daiyuanm** [iscyy/ultralyticsPro#14]
Thanks for your work, just amazing!

I noticed that there are many models that use Hornet, but there is no corresponding horblock in common.py and yolo.py. Looking forward to further improvement of the project.

By the way, I currently need a model with the best trade-off between mAP and latency. Since I currently don't have enough equipment and time to try all the models, do you have any recommendations?

**Comment by iscyy** [comment]
thanks, This part of the code has been updated
Second, What is the approximate range of mAP and latency requirements?

**Comment by daiyuanm** [comment]
> 

I did some experiments on my own dataset (5854 images) and here are some of the results (on TITAN Xp):

YOLOv5-x   mAP: 89.4 FPS: 21.65
YOLOv5-s   mAP: 87.9 FPS: 63.29

my model (based on YOLOv5-m, ViT, CBAM, and other tricks) mAP: 89.7 FPS: 36.1

I want to continue improving my model to have a better balance. There are many points worth trying in your project, but I have limited resources to experiment much at the moment. During your training, did you find any improvements that would give a better balance or individual improvements for mAP or FPS?

**Comment by daiyuanm** [comment]
> thanks, This part of the code has been updated Second, What is the approximate range of mAP and latency requirements?



I did some experiments on my own dataset (5854 images) and here are some of the results (on TITAN Xp):

YOLOv5-x mAP: 89.4 FPS: 21.65
YOLOv5-s mAP: 87.9 FPS: 63.29

my model (based on YOLOv5-m, ViT, CBAM, and other tricks) mAP: 89.7 FPS: 36.1

I want to continue improving my model to have a better balance. There are many points worth trying in your project, but I have limited resources to experiment much at the moment. During your training, did you find any improvements that would give a better balance or individual improvements for mAP or FPS?
